### PR TITLE
Improve English in Kotlin/Native as an Apple framework tutorial

### PR DESCRIPTION
**Pull request by Chozzle**
Makes it slightly easier to read

### CODE REVIEW

File: docs/topics/apple-framework.md [hunk 1]
```diff
@@ -452,19 +452,19 @@ the relative path to the compiled framework.
 
 ### Xcode for iOS targets
 
-First, you need to include the compiled framework into the Xcode project. For
-this, add the framework to the `Embedded Binaries` block of the `General` section of
-the *target* configuration page. 
+First, you need to include the compiled framework in the Xcode project. To do
+this, add the framework to the `Frameworks, Libraries, and Embedded Contentn` section of the `General`
```
Comment (Chozzle) [inline docs/topics/apple-framework.md:456]:
```suggestion
this, add the framework to the `Frameworks, Libraries, and Embedded Content` section of the `General` 
```

File: docs/topics/apple-framework.md [hunk 2]
```diff
@@ -452,19 +452,19 @@ the relative path to the compiled framework.
...
-this, add the framework to the `Embedded Binaries` block of the `General` section of
-the *target* configuration page. 
+First, you need to include the compiled framework in the Xcode project. To do
+this, add the framework to the `Frameworks, Libraries, and Embedded Content` section of the `General` 
+tab of the *target* configuration page. 
 
-The second step is to then include the framework path into the `Framework Search Paths` block
-of the `Build Settings` section of the *target* configuration page. It is possible to use `$(PROJECT_DIR)`
+The second step is to then include the framework path into the `Framework Search Paths` section
+of the `Build Settings` tab of the *target* configuration page. It is possible to use `$(PROJECT_DIR)`
```
Comment (Chozzle) [inline docs/topics/apple-framework.md:460]:
```suggestion
of the `Build Settings` tab of the *target* configuration page. It is possible to use the `$(PROJECT_DIR)`
```

File: docs/topics/apple-framework.md [hunk 3]
```diff
@@ -452,19 +452,19 @@ the relative path to the compiled framework.
 
 ### Xcode for iOS targets
 
-First, you need to include the compiled framework into the Xcode project. For
-this, add the framework to the `Embedded Binaries` block of the `General` section of
-the *target* configuration page. 
+First, you need to include the compiled framework in the Xcode project. To do
+this, add the framework to the `Frameworks, Libraries, and Embedded Content` section of the `General` 
+tab of the *target* configuration page.
```
Comment (koshachy) [inline docs/topics/apple-framework.md]:
```suggestion
tab of the **target** configuration page. 
```

**Review by koshachy — APPROVED** [review]

**Comment by koshachy** [comment]
Hey, thanks for your PR!

**Pull request merged**